CONTRIBUTING.md

```diff
@@ -8,24 +8,59 @@ After following the steps in the [Contribution Guide](http://docs.mattermost.com
 
 The one exception may be around release time, where the review process may take longer as the team focuses on our [release process](https://docs.mattermost.com/process/release-process.html). 
 
-#### `Stage 1: PM Review`
+#### PR submitted
+Dev applies labels and alerts PMs that there is a PR awaiting review by posting in the [PM/Docs PR Review channel](https://pre-release.mattermost.com/core/channels/pmdocs-pr-review-pub)
```

Comment:
By "Dev" I assume you mean "Someone on the Developer Team"? Or do you mean "A developer"?